src/functions/index.ts

```diff
@@ -128,9 +128,11 @@ function createComputeFunction(
 
     let vectorArgsType: VectorArgType[] | undefined = undefined;
 
+    const getArgToFocus = argTargeting(descr, args.length);
```

Comment:
it looks like the evaluation is significantly slower on this branch :/ (+10% to evaluate the large formula dataset)

before | after
-- | --
314.2ms | 346.6ms


I didn't check where exactly it comes from, but most probably this line here isn't free and it's executed *a lot* of times

Reply:
thx for measurements